examples/replica/outputs.tf

```diff
@@ -19,5 +19,5 @@ output "version" {
 
 output "replica_version" {
   description = "Postgresql read-only replica instance version"
-  value       = module.read_only_replica_postgresql_db.version
+  value       = module.read_only_replica_postgresql_db[0].version
```

Comment:
Given that the version is configurable through a single parameter for master and replicas, I don't think it would be needed to output. But if returning the version of replicas for a checking purpose, shouldn't it return the value for all the replicas instead of the previous one?

Reply:
Removing this as it is not needed.